src/components/FeedbackSticky/index.tsx

```diff
@@ -0,0 +1,203 @@
+import { useCallback, useEffect, useRef, useState } from 'react';
+import {
+  FeedbackContainer,
+  VoteButton,
+  VoteButtonReplace,
+  VoteButtonsContainer,
+  FeedbackText,
+  Divider,
+  ButtonStyles
+} from './styles';
+import { trackFeedbackSubmission } from '../../utils/track';
+import { Icon, Button } from '@cloudscape-design/components';
+
+enum FeedbackState {
+  START = 'START',
+  UP = 'UP',
+  DOWN = 'DOWN',
+  HIDDEN = 'HIDDEN'
+}
+
+type Feedback = {
+  vote: boolean;
+  page_path: string;
+  id?: string;
+  comment?: string;
+};
+
+export default function Feedback({ footerRef }) {
+  const [state, setState] = useState<FeedbackState>(FeedbackState.START);
+  const containerRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    const footer = footerRef.current;
+
+    if (typeof window !== 'undefined') {
+      window.addEventListener('touchmove', hideFeedback);
+      window.addEventListener('scroll', hideFeedback);
+
+      let prevScrollPos = window.scrollY;
+
+      // eslint-disable-next-line no-inner-declarations
+      function hideFeedback() {
+        // Scroll variables and calculations
+        const currPos = window.scrollY,
+          footerVisible =
+            document.body.scrollHeight - footer.offsetHeight + 50 <=
+            currPos + window.innerHeight,
+          up = prevScrollPos >= currPos,
+          down = prevScrollPos < currPos,
+          container = containerRef.current;
+
+        if (
+          container &&
+          ((up && footerVisible) || down) &&
+          !container.classList.contains('close')
+        ) {
+          container.classList.remove('slideIn'),
```

Comment:
a little nit picky but is there a reason here to use commas instead of semi-colons for these statements?